pkg/sql/catalog/lease/descriptor_version_state.go

```diff
@@ -115,7 +124,18 @@ func (s *descriptorVersionState) hasExpired(timestamp hlc.Timestamp) bool {
 // hasExpired checks if the descriptor is too old to be used (by a txn
 // operating) at the given timestamp.
 func (s *descriptorVersionState) hasExpiredLocked(timestamp hlc.Timestamp) bool {
-	return s.mu.expiration.LessEq(timestamp)
+	// Pick the longest expiration between the session and
```

Comment:
Very similar comment to the last PR 😅 , place the description of the condition ahead of the action description when the comment is outside of the `if` block.